### PR TITLE
Support for AArch64 macOS added

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/992

Updated the 0.35.0 release note with support for AArch64

[skip ci]

Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by pshipton** [comment]
https://eclipse-openj9.github.io/openj9-docs/openj9_support/ needs updates for AArch64 macOS as well. jdk11 and later is supported (not jdk8).

The compile OS is 11.5.2, with clang/xcode 13.0.

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have added the operating system and compiler level information of macOS AArch64 as a part of [#992](https://github.com/eclipse-openj9/openj9-docs/issues/992) ([PR#1010](https://github.com/eclipse-openj9/openj9-docs/pull/1010)).

File: doc/release-notes/0.35/0.35.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ OpenJ9 release 0.35.0 supports OpenJDK 8, 11, 17.
 
 All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests provided by Adoptium.
 
+AArch64 macOS&reg; is now a fully supported, production-ready target for OpenJDK 17.
```
Comment (pshipton) [inline doc/release-notes/0.35/0.35.md:35]: It's supported for OpenJDK 11 and later, which means 11 and 17 in the release, but also 19/20/etc when these versions are released later.

**Comment by pshipton** [comment]
- the "Early access release for Apple Silicon macOS" limitation should be removed
- we should refer to it as "Apple Silicon macOS" and not AArch64. In the User Guide as well.

@knn-k fyi

**Comment by knn-k** [comment]
Please remove the lines 123-128 in the original document as @pshipton mentioned.
Please also change "Apple Silicon macOS (early access)" in L.133 to "Apple Silicon macOS".  The restriction for the compressed references mode remains in v0.35.0.

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have added the support for AArch64 macOS in the table for notable changes. If the changes are ok, please merge this change. Thanks!

**Comment by pshipton** [comment]
Merging is waiting on final testing to confirm AArch64 Mac is leaving EA status.

**Comment by pshipton** [comment]
The current target to complete testing for AArch64 Mac is Nov 15.

**Review by pshipton — APPROVED** [review]

**Pull request merged**